mne/io/base.py

```diff
@@ -1649,6 +1649,8 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
         overwrite : bool
             If True, the destination file (if it exists) will be overwritten.
             If False (default), an error will be raised if the file exists.
+            If True and destination file exists data must be preloaded to
+            overwrite existing file.
```

Comment:
This still doesn't sound correct. This makes it sound like data must be reloaded to overwrite any existing file. It only needs to be preloaded if overwriting the same file that was loaded.

Reply:
How about...
>If True, the destination file (if it exists) will be overwritten.
If False (default), an error will be raised if the file exists.
To overwrite data using the same file name, the raw data should be preloaded during the reading operation.

Reply:
I would use @larsoner wording, smth like: 
> *to overwrite the original file (the same one that was loaded) ...*